resources/views/dielo.blade.php

```diff
@@ -158,6 +158,16 @@ class="sr-only">{{ trans('dielo.item_next-work') }}</span></a>
                                     </td>
                                 </tr>
                             @endif
+                            @if (!empty($item->object_type))
+                                <tr>
+                                    <td class="atribut">{{ trans('dielo.item_attr_object_type') }}:</td>
+                                    <td>
+                                        @foreach ($item->object_types as $object_type)
+                                        <a href="{{ URL::to('katalog?object_type=' . $object_type) }}">{{ $object_type }}</a><br>
```

Comment:
I think it might be nice to replace `URL::to` with `route()` calls. Slightly more verbose but also more robust.

```suggestion
                                        <a href="{{ route('frontend.catalog.index', ['object_type' => $object_type]) }}">{{ $object_type }}</a><br>
```

Reply:
It was just copy pasted from above. Do you think I should replace them all?